conf/defaults.ini

```diff
@@ -1133,3 +1133,34 @@ default_baselayer_config =
 
 # Enable or disable loading other base map layers
 enable_custom_baselayers = true
+
+#################################### Dashboard previews #####################################
+[dashboard_previews.images]
+# Width of each image. Default is 320.
+width =
```

Comment:
we don't want end user control over this (yet) because the UX will need to depend on it